src/components/MAPI/__tests__/utils.ts

```diff
@@ -15,4 +15,70 @@ describe('mapi::utils', () => {
       }
     });
   });
+
+  describe('determineRandomAZs', () => {
+    it('returns a collection of random availability zones', () => {
```

Comment:
This test tries to get `3` unique arrays of `2` AZs from the available `['1', '2', '3', '4', '5']` list of AZs. It runs that 50 times, and expects that at least 50% of the time the returned AZs are random. 

This is so we wouldn't have a flaky test that makes us re-run CI every 5 mins.